tests/fake_filesystem_unittest_test.py

```diff
@@ -57,10 +57,10 @@ class TestPyfakefsUnittest(TestPyfakefsUnittestBase):  # pylint: disable=R0904
     def test_file(self):
         """Fake `file()` function is bound"""
         self.assertFalse(os.path.exists('/fake_file.txt'))
-        with file('/fake_file.txt', 'w') as f:
+        with open('/fake_file.txt', 'w') as f:
             f.write("This test file was created using the file() function.\n")
         self.assertTrue(self.fs.exists('/fake_file.txt'))
-        with file('/fake_file.txt') as f:
+        with open('/fake_file.txt') as f:
             content = f.read()
```

Comment:
These are actually wanted - it explicitely tests that `file` for Python2 id faked.
The rest looks good!